src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.h

```diff
@@ -6,9 +6,12 @@
 
 struct EnWeatherTag;
 
+typedef void (*EnWeatherTagActionFunc)(struct EnWeatherTag*, GlobalContext*);
+
 typedef struct EnWeatherTag {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x8];
+    /* 0x014C */ EnWeatherTagActionFunc actionFunc;
+    /* 0x0150 */ s16 unk_150; //unused
```

Comment:
If it's unused it should just be left as an unknown filler, since we can't even know what type it is.
```suggestion
    /* 0x0150 */ char unk_150[0x04];
```